packages/web-app-files/src/services/folder/legacy/loaderPersonal.ts

```diff
@@ -52,7 +55,7 @@ export class FolderLoaderLegacyPersonal implements FolderLoader {
 
         // fetch user quota
         ;(async () => {
-          const user = await client.users.getUser(ref.user.id)
+          const user = await client.users.getUser(router.currentRoute.params.storageId)
           store.commit('SET_QUOTA', user.quota)
         })()
```

Comment:
This seems wrong, afaict this would update the user specific quota when viewing someone else's personal space.

What happens, when we use a bookmark to enter oC Web and skipping the personal view? Does the quota get loaded at all (it's visible from the user menu)? This seems to the wrong place after all.

Reply:
Good point, will create a followup issue to query quota from the backend in some other place (maybe quota component in the topbar, maybe something that polls with low frequency, let's see...)

Reply:
created https://github.com/owncloud/web/issues/7093 to follow up on this